ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_creation.py

```diff
@@ -376,3 +376,42 @@ def test_jax_numpy_eye(
         k=k,
         dtype=dtypes[0],
     )
+
+
+# triu
+@handle_frontend_test(
+    fn_tree="jax.numpy.triu",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=1,
+        min_num_dims=2,
+        max_num_dims=5,
+        min_dim_size=1,
+        max_dim_size=5,
+    ),
+    k=helpers.ints(min_value=-10, max_value=10),
+)
+
+def test_numpy_triu(
```

Comment:
There is a lint error caused because there shouldn't be a blank line between the decorator and the function.

Reply:
Fixed✅